css/style.css

```diff
@@ -507,33 +505,149 @@ address {
     margin-top: 0em;
 }
 
+.floe .idrc-logo-container svg,
+.floe .hewlett-logo-container svg {
+    padding: 0.5em;
+    width: 100%;
+    height: auto;
+}
+
+/* IE-only hacks for better SVG sizing */
+
+/* Large, two column */
+@media all and (min-width: 64em) and (-ms-high-contrast:none)
```

Comment:
@waharnum this solution that uses both `active` and `none` seems to work.

https://paper-leaf.com/blog/2014/09/targeting-ie-10-11-browsers-css/